Userland/Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -873,20 +873,31 @@ ThrowCompletionOr<PropertyKey> Value::to_property_key(VM& vm) const
 ThrowCompletionOr<i32> Value::to_i32_slow_case(VM& vm) const
```

Comment:
Perhaps this should also have a link to https://tc39.es/ecma262/#sec-toint32 ?
Since it might otherwise be somewhat unclear where these steps come from.

Reply:
Done